rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/tools/osb/mbean/ObjStoreBrowserLRATest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Red Hat
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package io.narayana.lra.coordinator.tools.osb.mbean;
+
+import com.arjuna.ats.arjuna.common.Uid;
+import com.arjuna.ats.arjuna.common.recoveryPropertyManager;
+import com.arjuna.ats.arjuna.tools.osb.mbean.ActionBean;
+import com.arjuna.ats.arjuna.tools.osb.mbean.LogRecordWrapper;
+import com.arjuna.ats.arjuna.tools.osb.mbean.OSEntryBean;
+import com.arjuna.ats.arjuna.tools.osb.mbean.ObjStoreBrowser;
+import com.arjuna.ats.arjuna.tools.osb.mbean.UidWrapper;
+import com.arjuna.ats.internal.arjuna.recovery.RecoveryManagerImple;
+import io.narayana.lra.coordinator.domain.model.LRAParticipantRecord;
+import io.narayana.lra.coordinator.domain.model.LongRunningAction;
+import io.narayana.lra.coordinator.internal.Implementations;
+import io.narayana.lra.coordinator.internal.LRARecoveryModule;
+import org.eclipse.microprofile.lra.annotation.LRAStatus;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.net.URI;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class ObjStoreBrowserLRATest {
+    private RecoveryManagerImple recoveryManager;
+
+    @Before
+    public void setUp() {
+        recoveryPropertyManager.getRecoveryEnvironmentBean().setRecoveryBackoffPeriod(1);
+        Implementations.install();
+        recoveryManager = new RecoveryManagerImple(false);
+        recoveryManager.addModule(new LRARecoveryModule());
+    }
+
+    @After
+    public void tearDown () {
+        recoveryManager.removeAllModules(false);
+        recoveryManager.stop(false);
+        Implementations.uninstall();
+    }
+
+    @Test
+    public void lraMBean() throws Exception {
```

Comment:
Remark: it is a good test. In the future we should add testing for the `StateManager/BasicAction/LongRunningAction/Failed` record type.